### PR TITLE
Feature/assia/logique/5.5

### PR DESCRIPTION
**Pull request by AssiaRachdi**

### CODE REVIEW

File: stagiaires/assia/exercices/logique-programmation/ex5.5.txt [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    ALORS
+        limite <- longueur(text2)
+    SINON
+        limite <- longueur(text1)
+    FINSI
+
+    REPETER
+        SI caract(text1,i) < caract(text2, i)
+        ALORS
+            reponse <- 0
```
Comment (winstw) [inline stagiaires/assia/exercices/logique-programmation/ex5.5.txt:33]: reponse <- FAUX

File: stagiaires/assia/exercices/logique-programmation/ex5.5.txt [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        FINSI
+        SI caract(text1,i) > caract(text2, i)
+        ALORS
+            reponse <- 1
```
Comment (winstw) [inline stagiaires/assia/exercices/logique-programmation/ex5.5.txt:37]: reponse <- VRAI

File: stagiaires/assia/exercices/logique-programmation/ex5.5.txt [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        FINSI
+        i <- i + 1
+    JUSQU'A CE QUE (reponse = 1) OU (i > limite)
```
Comment (winstw) [inline stagiaires/assia/exercices/logique-programmation/ex5.5.txt:40]: ```JUSQU'A CE QUE reponse OU i > limite```

File: stagiaires/assia/exercices/logique-programmation/ex5.5.txt [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+- Algorithme au seul niveau primaire.*
+
+Debut Exercice
+
+    VARIABLE LOCALES :
+        CONST text1 :T <- ""
+        CONST text2 :T <- ""
+        CONST limite :N <- 0
+        VAR i :N <- 1
+        reponse :B <- 0
```
Comment (winstw) [inline stagiaires/assia/exercices/logique-programmation/ex5.5.txt:15]: VAR reponse : B <- FAUX

File: stagiaires/assia/exercices/logique-programmation/ex5.5.txt (hunk 1 above)
Comment (AssiaRachdi) [reply stagiaires/assia/exercices/logique-programmation/ex5.5.txt:33]: done

File: stagiaires/assia/exercices/logique-programmation/ex5.5.txt (hunk 3 above)
Comment (AssiaRachdi) [reply stagiaires/assia/exercices/logique-programmation/ex5.5.txt:40]: done

File: stagiaires/assia/exercices/logique-programmation/ex5.5.txt (hunk 1 above)
Comment (AssiaRachdi) [reply stagiaires/assia/exercices/logique-programmation/ex5.5.txt:33]: done

**Pull request merged**